### PR TITLE
Onboarding wizard

### LINKED ISSUE #530: Onboarding Wizard
I have completely re-done the experience so I am updating the original issue.

This is a two-step setup. Please view Loom video at the bottom of the page

[Figma ](https://www.figma.com/file/WE9qxn4dt8TEmHa7JIhNi6/Style-Kits-Remix?node-id=713%3A2500)
onboarding wizard

Let me know your thoughts.

### Loom video:
https://www.loom.com/share/19f45a621ca14dc1b7fee63bacca3daa

### LINKED ISSUE #622: Merge CSS files for library
Multiple CSS files are applied to library with basically no conditions, let's merge them. And hopefully clean it up as well.

### PR DESCRIPTION
**Pull request by lushkant**
Addresses #530

### CODE REVIEW

File: inc/class-onboarding.php [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+	/**
+	 * Imports base style kit.
+	 *
+	 * @return bool
+	 */
+	private function import_base_kit() {
+		$kit = array(
+			'id'             => 6491,
+			'title'          => 'Style Kit: Base',
+			'image'          => 'https://preview.analogwp.com/designsystem/wp-content/uploads/sites/26/2022/11/base.svg',
```
Comment (mauryaratan) [inline inc/class-onboarding.php]: Lets not add a remote image here, since that can be removed at any point.

Should be saved locally.

Comment (lushkant) [reply inc/class-onboarding.php]: Resolving this, as it is already being done at import.

**Comment by mauryaratan** [comment]
@lushkant Can you please fix the merge conflicts?

**Comment by lushkant** [comment]
> @lushkant Can you please fix the merge conflicts?

@mauryaratan done!

**Comment by lushkant** [comment]
@mauryaratan please let me know if there is any other issue so that we can merge it.

**Pull request merged**